pyproject.toml

```diff
@@ -2,3 +2,4 @@
 requires = ["setuptools>=42", "wheel", "setuptools_scm[toml]>=3.4"]
```

Comment:
```suggestion
requires = ["setuptools>=42", "wheel", "setuptools_scm[toml]>=3.4"]
build-backend = "setuptools.build_meta"
```

Reply:
Yup. I found that :facepalm: 

Thanks!

Reply:
The thing is that some frontends (like pip) implement a fallback and set it to `setuptools.build_meta.__legacy__` and others don't.